api/src/main/java/net/kyori/adventure/text/event/HoverEvent.java

```diff
@@ -209,6 +209,19 @@
     return new HoverEvent<>(Action.SHOW_ENTITY, entity);
   }
 
+  /**
+   * Creates a hover event that shows an achievement on hover.
+   *
+   * @param value the achievement value
+   * @return a hover event
+   * @since 4.13.1
```

Comment:
```suggestion
   * @since 4.14.0
```

(we wouldn't add new API in a semver patch release anyways)